module_end_wildfire_smoke.r

```diff
@@ -152,4 +156,25 @@ endWildfireSmoke <- function(input, output, session){
       id <- showNotification("Markdown file moved to outputs/qmd directory.")
     }
   })
+  ## Download files
+  
+  output$download_report <- downloadHandler(
+    filename = function() {
+      sprintf("%s_wildfire_smoke_end.zip", Sys.Date())
+    },
+    content = function(file) {
+      # Build file paths correctly using sprintf()
+      files_to_zip <- c(
+        sprintf("outputs/qmd/%s_wildfire_smoke_end.md", Sys.Date()),
+        sprintf("outputs/rnw/%s_wildfire_smoke_end.pdf", Sys.Date())
+      )
+      
+      # Check they exist (optional but good practice)
+      files_to_zip <- files_to_zip[file.exists(files_to_zip)]
```

Comment:
What happens if the files don't exist? Not sure what this check will do if it fails.